PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue

```diff
@@ -95,8 +95,8 @@
       <v-col class="d-flex" cols="5" sm="5">
         <!-- Current Photo -->
         <v-file-input
-          :rules="rules"
           accept="image/png, image/jpeg, image/bmp"
+          :rules="[(v) => !v || v.size < 2097152 || 'Image size should be less than 2 MB!']"
```

Comment:
I'm assuming `v.size < 2097152` is the actual file size check. What's that number represent? Will this be consistent across different platforms and OS' (e.g. Apple vs. Microsoft)?

Reply:
The number represents bytes. It can be different across operating systems, however I don't think we'll have issues as I'm using the larger interpretation.

Reply:
Sounds good